packages/webapp/src/store/api/apiSlice.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright 2023-2024 LiteFarm.org
```

Comment:
I think this can be `2024`.

Reply:
I wanted to give credit to @antsgar for the code she wrote in 2023 (upon which this was based!) 😄 But I guess since it never got merged, this is a bit confusing.